lib/ui/views/onboarding_view.dart

```diff
@@ -14,68 +16,82 @@ class _OnBoardingViewState extends State<OnBoardingView> {
   void initState() {
     super.initState();
     currentIndexPage = 0;
-    pageLength = 3;
+    pageLength = 6;
   }
 
   @override
   Widget build(BuildContext context) {
-    double marginFromSafeArea = 24;
-    var heightOfScreen =
-        MediaQuery.of(context).size.height - marginFromSafeArea;
     return Scaffold(
       body: SafeArea(
-          child: Container(
-        child: Column(
-          children: <Widget>[
-            Container(
-              height: heightOfScreen * 0.6,
-              child: PageView(
-                children: <Widget>[
-                  _buildWalkThrough(),
-                  _buildWalkThrough(),
-                  _buildWalkThrough()
-                ],
-                onPageChanged: (value) {
-                  setState(() => currentIndexPage = value.toDouble());
-                },
-              ),
-            ),
-            Container(
-              height: heightOfScreen * 0.1,
-              child: DotsIndicator(
-                dotsCount: pageLength,
-                position: currentIndexPage,
-                decorator: DotsDecorator(
-                  color: Colors.blue,
-                  activeColor: Colors.red,
+        child: Container(
+          child: Column(
+            children: <Widget>[
+              Flexible(
+                child: Container(
+                  child: PageView(
+                    children: <Widget>[
+                      _buildWalkThrough(context, 'undraw_Hello_qnas.svg',
+                          "Welcome to DigiPol! Where digital direct democracy starts."),
+                      _buildWalkThrough(context, 'undraw_process_e90d.svg',
+                          "DigiPol collates all the current bills in the Australian goverment for you."),
+                      _buildWalkThrough(context, 'undraw_voting_nvu7.svg',
+                          "So you can stay informed & make your opinion known when it matters most."),
+                      _buildWalkThrough(context, 'undraw_new_ideas_jdea.svg',
+                          "You can even vote on current issues created by the community or submit your own (in development)."),
```

Comment:
It's not in development. We don't even know HOW we'll do it. We need terms of service, research into user-content in app, moderation standards, etc, etc, etc, etc, etc

Unless you are going to take those responsibilities, don't put the text down. It doesn't matter here, just say:

"we're hoping to add this in the future" or something!